Tests/DatadogTests/Datadog/URLSessionAutoInstrumentation/DDURLSessionDelegateAsSuperclassTests.swift

```diff
@@ -6,6 +6,18 @@
 
 import XCTest
 import Datadog
+import DatadogObjc
+
+@objc
+public class SubDDNSURLSessionDelegate: DDNSURLSessionDelegate {
+    override public func urlSession(_ session: URLSession, task: URLSessionTask, didCompleteWithError error: Error?) {
+        super.urlSession(session, task: task, didCompleteWithError: error)
+    }
+
+    override public func urlSession(_ session: URLSession, task: URLSessionTask, didFinishCollecting metrics: URLSessionTaskMetrics) {
+        super.urlSession(session, task: task, didFinishCollecting: metrics)
+    }
+}
```

Comment:
I think we may also want to instantiate this class as we do for `DDURLSessionDelegate` subclass:
```swift
_ = SubDDURLSessionDelegate()
```